uint32 seq		# sequence ID: consecutively increasing ID 
time stamp		# Two-integer timestamp of data
string agent1_id	# First robot data is associated with
float64 x1		# X-coordinate of first robot
float64 y1		# Y-coordinate of first robot
float64 z1		# Z-coordinate of first robot
float64 yaw1		# Yaw (heading) of first robot
string agent2_id	# Second robot data is associated with
float64 x2		# X-coordinate of second robot
float64 y2		# Y-coordinate of second robot
float64 z2		# Z-coordinate of second robot
float64 yaw2		# Yaw (heading) of second robot
